lighthouse-core/fraggle-rock/config/config.js

```diff
@@ -108,7 +108,9 @@ function resolveArtifactsToDefns(artifacts, configDir) {
       throw new Error(`${gatherer.instance.name} gatherer does not support Fraggle Rock`);
     }
 
-    /** @type {LH.Config.ArtifactDefn<LH.Gatherer.DependencyKey>} */
+    /** @type {LH.Config.AnyArtifactDefn} */
+    // @ts-expect-error - Typescript can't validate the gatherer and dependencies match
```

Comment:
this is the only place I couldn't figure out how to solve.

🤓 🔫  @brendankenny if you're interested :)

the only thing I could think was to *re-validate* that `resolveArtifactDependencies` matches outside the function, but that seemed excessive